trunner/harness/plo.py

```diff
@@ -72,13 +72,16 @@ def __init__(self, dut: Dut):
     def enter_bootloader(self):
         """Interrupts timer counting to enter plo."""
 
+        # If armv7a9-zynq7000-zedboard is flashed first time (or after full flash0 erase) it will throw error magic
+        # number mismatch, to avoid interrupt and flash it anyway those are taken as false negative and execute further
```

Comment:
maybe it would be better to write:
magic number mismatch error, instead error magic number mismatch